README.md

```diff
@@ -0,0 +1,186 @@
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a id="getting-started"></a>
+<br>
+
+### Prerequisites <a id="prerequisites"></a>
+
+In order to reproduce the webdev-inst on your local machine and be able to make changes for your own project you need the following tools:
+  - Visual Studio Code
+  - <a href="https://git-scm.com/">git-scm</a>
+  - A Github account
+
+### Setup
+
+  1. Make a new directory and go inside the directory using cd command in terminal
+  2. Use "git" to clone this repository into your local drive:
+      >```$ git clone https://github.com/dev89s/webdev-inst.git```
+
+### Usage
+
+  - To be able to see the project's current state use this command:
+    > ```$ git branch checkout dynamic-page```
+  - Feel free to change the code and make any adjustment to the design and once you are satisfied with your work you can make a new directory and copy your files there.
+
+### Deployment <a id="deployment"></a>
+
+You can deploy this project by opening a new github repo and initialize the local git repo and connect it through: ```$ git remote add origin main [your github repo link]``` command.
+Then you can make a pull requrest and then if you go to the repository in your account, you'll see the pull requrest prompt appear at the top and the you can make a pull request and observe the Github Actions checking Linters.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a id="authors"></a>
+
+👤 **Sasan Moshir**
+
+- GitHub: [@dev89s](https://github.com/dev89s)
+- Twitter: [@dev89_s](https://twitter.com/dev89_s)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a id="future-features"></a>
+
+- **Updated design with cleaner design**: the desgin will improve significantly. 
+- **Implement other pages**: Other pages will be added to the template later.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a id="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/dev89s/portfolio/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a id="support"></a>
+
+If you like this project consider giving a star to the project and if you want to go furthur make a donation to any one the top 8 charities in Hawaii islands at the link below:
+
+-   [Hawaii Donation](https://www.hawaii-guide.com/the-top-8-charities-you-should-donate-to-in-hawaii)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a id="acknowledgements"></a>
+
+I would like to thank all the coding partners that helped me through building this example.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
```

Comment:
- [ ] You did a wonderful job with creating the README file :+1:, but kindly don't forget to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). It should be done in the 'Acknowledgements' section